runconfig/parse.go

```diff
@@ -441,6 +441,8 @@ func ParseRestartPolicy(policy string) (RestartPolicy, error) {
 			}
 
 			p.MaximumRetryCount = count
+		} else if len(parts) > 2 {
```

Comment:
@coolljt0725 can you move this at the start of this case and return immediately instead of doing this else if? :)